Config.psm1

```diff
@@ -151,6 +151,10 @@ function Get-AvailableConfigOptions {
           "Description" = "If set to true and the target image format is QCOW2, the image conversion will
                            use qemu-img built-in compression. The compressed qcow2 image will be smaller, but the conversion
                            will take longer time."},
+        @{"Name" = "sdelete_cleanup"; "DefaultValue" = $false; "AsBoolean" = $true;
```

Comment:
you can change this config name and description according to @alexpilotti suggestion. In the future, sdelete functionality can be replaced with n existing windows tool or an open source one.